### ISSUE TITLE
Missing icon: printer

### ISSUE THREAD

**Issue by Feichtmeier** [ubuntu/yaru#1057]
![image](https://user-images.githubusercontent.com/15329494/50366632-14d30200-057b-11e9-8b5e-a0883a3158ce.png)

@ubuntujaggers @eaglersdeveloper :)

**Comment by ubuntujaggers** [comment]
Aha, this will either be a fun one or a very hard one  😛 

I've got a quiet Christmas period so should be able to get a lot of Inkscape done next week and the week after, so I'll add this to the list.  I've also got battery icon to look at, let me know if there's any I've forgotten.

**Comment by madsrh** [comment]
@Feichtmeier So can we close this one?

**Comment by Feichtmeier** [comment]
No, there are generic printer icons for most printers. Only a few have that special icons :)

**Comment by ubuntujaggers** [comment]
I've been trying this one today, so far it's proving tough but I'll keep working on it.

**Comment by ubuntujaggers** [comment]
I feel like I'm making progress now - how about this (I've only done 48px so far):

![image](https://user-images.githubusercontent.com/38893390/50539673-fcb34200-0b7b-11e9-93df-157cad8ce411.png)

The body of the printer has the same size/shape/position/gradient as the keyboard icon for maximum consistency.

Here it is with some other devices at 48px for comparison:

![image](https://user-images.githubusercontent.com/38893390/50539683-1a80a700-0b7c-11e9-94b1-4971c0189e59.png)

![image](https://user-images.githubusercontent.com/38893390/50539687-2bc9b380-0b7c-11e9-9880-d474d9aa7213.png)

![image](https://user-images.githubusercontent.com/38893390/50539697-4b60dc00-0b7c-11e9-8353-5546e0824705.png)


![image](https://user-images.githubusercontent.com/38893390/50539674-0177f600-0b7c-11e9-997c-c94c042b2c31.png)

**Comment by Feichtmeier** [comment]
Sexy!

**Comment by ubuntujaggers** [comment]
Brilliant - I'll do the remaining sizes :)

**Comment by ubuntujaggers** [comment]
![image](https://user-images.githubusercontent.com/38893390/50550779-33698500-0c6e-11e9-96dd-f30822aadfa2.png)

??

**Comment by clobrano** [comment]
It looks very nice, good job!

**Comment by clobrano** [comment]
Just one note, have you already considered to make the top border a bit less thick?
I am talking about the mouth of printer, where the paper exits. In the small sizes, it is fine, but at the biggest one a bit less

**Comment by ubuntujaggers** [comment]
I will certainly have a go! Do you mean the horizontal dark hairline?

**Comment by clobrano** [comment]
This border

![image](https://user-images.githubusercontent.com/2883614/50551619-1d66bf00-0c84-11e9-871c-e35c89ae01be.png)

I know I am nitpicking, but usually printers have a very thin border there

**Comment by madsrh** [comment]
I like it @ubuntujaggers but isn't there something odd about the perspective/angle? 
I'm not sure if it's the same thing as @clobrano is referring to. The small icons looks perfect, but the large icon looks like I'm looking down on the printer. Does this make sense?

**Comment by ubuntujaggers** [comment]
> I like it @ubuntujaggers but isn't there something odd about the perspective/angle?
> I'm not sure if it's the same thing as @clobrano is referring to. The small icons looks perfect, but the large icon looks like I'm looking down on the printer. Does this make sense?

Yes there is  :)  Not sure why it looks okay in the smaller 48px but not the full size one...  but I agree with you, the biggest one looks odd 🤔 

Okay, I've tried three things.  No. 1 is making the top edge of the "mouth" thinner as per @clobrano's suggestion:

![image](https://user-images.githubusercontent.com/38893390/50557968-f4751700-0ce1-11e9-9026-b1f555b18cb2.png)

No. 2 is making the top highlight on the mouth less pronounced in the largest size, which might make the (odd) perspective jump out less:

![image](https://user-images.githubusercontent.com/38893390/50558009-4027c080-0ce2-11e9-9ab7-cc9f0dcedfd1.png)

No. 3 is my least favourite - I actually tried to go along with the suggested perspective a bit more, but I don't think the result is very Suru, and I think the foreshortening would probably have to be more pronounced for it to look right (also, it makes the flat lights look weird because they aren't drawn from the same angle):

![image](https://user-images.githubusercontent.com/38893390/50558054-998fef80-0ce2-11e9-83e5-7349e69597b8.png)

Out of the three, my favourite is the middle one (no. 2), but I'll keep playing with the svg and see what I can achieve, comments welcome as always!  :)

**Comment by Feichtmeier** [comment]
I'd prefer the 2) one

**Comment by clobrano** [comment]
I agree, #2 looks better for me as well

**Comment by ubuntujaggers** [comment]
Magic, I'll push as soon as I'm on my main laptop  👍

**Comment by Feichtmeier** [comment]
I need to make some symlinks t make it work

**Comment by eaglersdeveloper** [comment]
You can close this issue.
